// [[Reason: Function declarations are not fixed]] ////

//// [tests/cases/compiler/declarationEmitExpandoPropertyPrivateName.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -5,8 +5,14 @@
 }
 export declare function f(): I;
 export {};
 //# sourceMappingURL=a.d.ts.map
+//// [b.d.ts]
+export declare function q(): void;
+export declare namespace q {
+    var val: I;
+}
+//# sourceMappingURL=b.d.ts.map
 /// [Errors] ////
 
 b.ts(5,14): error TS2304: Cannot find name 'I'.
 b.ts(5,14): error TS4025: Exported variable 'val' has or is using private name 'I'.
